test/e2e/kubevirt.go

```diff
@@ -353,6 +268,59 @@ passwd:
 			return fr.ClientSet.NetworkingV1().NetworkPolicies(namespace).Create(context.TODO(), policy, metav1.CreateOptions{})
 		}
 
+		checkEastWestTraffic = func(vmi *kubevirtv1.VirtualMachineInstance, podIPsByName map[string][]string, stage string) {
+			polling := time.Second
+			timeout := 10 * time.Second
+			for podName, podIPs := range podIPsByName {
+				for _, podIP := range podIPs {
+					output := ""
+					Eventually(func() error {
+						var err error
+						output, err = kubevirt.RunCommand(vmi, fmt.Sprintf("curl http://%s", net.JoinHostPort(podIP, "8000")), polling)
```

Comment:
you're again passing polling. If you check the `kubevirt.RunCommand` function, it is supposed to receive the timeout, not the polling interval:
https://github.com/ovn-org/ovn-kubernetes/blob/51b2596f711a182b0f09182d30c4321529671d43/test/e2e/kubevirt/console.go#L84

Reply:
I am leaving this exactly as it is at `origin/master` make sense to pass polling so we "break" RunCommand inside eventually and it retries, if we don't do that it will not retry (maybe curl is already retrying, don't know).

Reply:
it's very different to wait for 15 seconds than wait for 1 minute ... 

anyway ... whatever. if you want to wait for 1 second, and it just works, it's fine by me.

Reply:
we wait for 1 minute at Eventually not at RunCommand.